modules/test/playwright/tests/workspaces/liferay-customer-workspace/pages/projectPaaSPage.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {CUSTOMER_SITE_FRIENLY_URL_PATH} from '../utils/constants';
+
+export class ProjectPaaSPage {
+	readonly finishActivationButton: Locator;
+	readonly heading: Locator;
+	readonly page: Page;
+	readonly projectName: Locator;
+	readonly projectNotActivatedTag: Locator;
+
+	constructor(page: Page) {
+		this.finishActivationButton = page.getByRole('button', {
+			name: 'Finish Activation Icon order-',
+		});
+		this.heading = page.getByRole('heading', {name: 'Activation Status'});
+		this.page = page;
+		this.projectName = page.getByRole('heading', {
```

Comment:
this specific project name isn't tied to this page so we dont need it here

Reply:
Done!